aiocache/decorators.py

```diff
@@ -75,6 +79,7 @@ def __init__(
 
         self._cache = cache
         self._serializer = serializer
+        self._namespace = namespace
```

Comment:
In some places, you've used `self.cache.namespace`, others `self._namespace`, do we need this extra attribute?

Reply:
Probably not.  Comments below suggest that it might not be needed after __init__() completes.